common/lib/xmodule/xmodule/course_module.py

```diff
@@ -294,6 +294,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_("Enter the url of the official Facebook page of the course."),
+        default="",
+        display_name=_("Course Facebook Page"),
+        scope=Scope.settings
+    )
```

Comment:
Yes, it is only for Facebook.  It is a URL that takes the user to the course' official facebook page.  That is my understanding.  You can get confirmation/clarification from Lou.